docusaurus/src/pages/index.js

```diff
@@ -37,39 +43,86 @@ export default function Home() {
-                <ul>
-                    <li>developers - yes!</li>
-                    <li>operators - yes!</li>
-                    <li>security folks - yes!</li>
-                </ul>
-                </p>
-                <p>OpenZiti provides all the tooling needed to run your own zero trust overlay network. If you would rather
-                    not maintain your own network, you can let NetFoundry manage the network for you. Head over to
-                    <Link to={"https://netfoundry.io/pricing/"}> the NetFoundry Console</Link> and learn more.</p>
+            <OpenZitiHorizontalSection style={{fontSize:"large", padding: "0px"}}>
+                <H2>Who is OpenZiti For?</H2>
+                <div style={{display: "flex", flexDirection: "column"}}>
+                    <div style={{display: "flex", flexDirection: "column"}}>
+                        <p>Software is for <Highlight>everyone</Highlight> - that's why its eating the world. Is OpenZiti for:
+                            developers, operators, security professionals, businesses, home users, you name it! OpenZiti
+                            provides all the tooling needed to run your own zero trust overlay network. If you would rather
+                            not maintain your own network, you can let NetFoundry manage the network for you. Head over to
+                            <Link to={"https://netfoundry.io/pricing/"}> the NetFoundry Console</Link> and learn more.</p>
+                    </div>
+                </div>
+            </OpenZitiHorizontalSection>
+            <OpenZitiHorizontalSection style={{fontSize:"large", padding: "0px"}}>
+                <div className={styles.whoziticontainer}>
+                    <div className={styles.whozitititlelogo}>
+                        <h2 style={{display:"flex", paddingRight: "10px", margin: "0px"}}>OpenZiti</h2>
+                        <img src={"/img/github.svg"} style={{display:"flex", alignItems:"center", height: "75px"}}/>
+                    </div>
+                    <div className={styles.whozitibullets}>
+                        <div style={{display:"flex", flexDirection: "column"}}>
+                            <p>Host everything yourself</p>
+                            <p>Some people learn best by <Highlight>doing</Highlight></p>
+                            <p>Fully configurable - you're running it all after all!</p>
+                        </div>
+                        <div className={styles.indexCtas}>
+                            <Link className="button button--primary" to="#build-a-network">
+                                <p>Get OpenZiti</p>
+                            </Link>
+                        </div>
+                    </div>
+                </div>
+                <div className={styles.whoziticontainer}>
+                    <div className={styles.whozitititlelogo}>
+                        <h2 style={{display:"flex", paddingRight: "10px", margin: "0px"}}>ZEDS</h2>
```

Comment:
We should spell-out "Ziti Edge Developer Sandbox" here because the name helps to explain in this context of choosing a hosting strategy to get started with Ziti, and because ZEDS doesn't yet have the recognition it needs to be meaningful all by itself.

Reply:
You could work in both like "Ziti Edge Developer Sandbox (ZEDS)" or as a sub-title like

### ZEDS
#### _Ziti Edge Developer Sandbox_